packages/dnb-eufemia/src/components/progress-indicator/ProgressIndicatorDocs.ts

```diff
@@ -39,8 +39,8 @@ export const ProgressIndicatorProperties: PropertiesTableProps = {
   },
   labelDirection: {
     doc: 'Sets the position of the label.',
-    type: ['"horizontal"', '"vertical"'],
-    defaultValue: '"horizontal"',
+    type: [`'horizontal'`, 'vertical`'],
```

Comment:
```suggestion
    type: [`'horizontal'`, `'vertical'`],
```

<img width="646" alt="Screenshot 2024-04-24 at 10 20 27" src="https://github.com/dnbexperience/eufemia/assets/1501870/2f1621fb-0488-4bb5-9081-966763e79924">